package.json

```diff
@@ -1,15 +1,6 @@
 {
   "name": "visual",
-  "scripts": {
-    "postinstall": "pbiviz update 1.6.0",
-    "pbiviz": "pbiviz",
-    "package": "pbiviz package",
-    "lint": "tslint -r \"node_modules/tslint-microsoft-contrib\" \"+(src)/**/*.ts\"",
-    "start": "pbiviz start"
-  },  
   "devDependencies": {
-    "tslint": "^4.4.2",
-    "tslint-microsoft-contrib": "^4.0.0",
-    "powerbi-visuals-tools": "1.6.2"
```

Comment:
Please revert it back as this line specifies a version of pbiviz that was used during development.
Please also increase version to 1.10.0

Reply:
done